man/machinery-inspect.1.md

```diff
@@ -48,6 +48,15 @@ trigger errors.
     PREREQUISITES for more information).
     To change the default-user use `machinery config remote-user=USER`
 
+  * `-p SSH-PORT`, `--port SSH-PORT` (optional):
+    Specifies the SSH port of the remote SSH server.
+
+  * `-k SSH-KEY`, `--key SSH-KEY` (optional):
+    Specifies the SSH private key what should be used to authenticate with the
+    remote SSH server. It's important that private keys with a passphrase
+    are already loaded by the `ssh-agent` so that it's possible to connect
+    not interactively. 
```

Comment:
Does this make sense? If ssh-agent already has the key then you don't need to provide it with an option, do you?